install_debian.sh

```diff
@@ -0,0 +1,23 @@
+#! /bin/bash
```

Comment:
In review of this, I think it would be worthwhile creating one script that can handle installation for all the supported distributions. There's a lead on that [here](https://unix.stackexchange.com/questions/46081/identifying-the-system-package-manager). This will keep the number of files in the repo from blowing up, but the big win being that it keeps the README.md simple. 

Let me know if you're interested in that.

Reply:
I agree, I've been thinking about this one. Maybe having something that detects the package manager and installs the packages based on a branching case-statement. Otherwise, having users enter "debian", "centos", etc. in for example "$ bash install_airplonk.sh debian" and having it interpret which package manager to then use from a case statement within the script. I haven't found a way to just install a list of packages it requires across a range of distros but will include an attempt at such a script.